src/elements/toast/Toast.js

```diff
@@ -0,0 +1,174 @@
+import { Component, Injectable, ApplicationRef, DynamicComponentLoader, ElementRef, Injector } from 'angular2/core';
+import { CORE_DIRECTIVES } from 'angular2/common';
+
+@Injectable()
+export class NovoToaster {
```

Comment:
Can we call this `ToastService`? and maybe move it into its own file `src/elements/toast/ToastService.js` with it's own stubbed spec. The documentation can still live in the same README